h/static/styles/partials-v2/_footer.scss

```diff
@@ -0,0 +1,65 @@
+// The amount of horizontal space between items in the footer.
+$spacing: 30px;
```

Comment:
Just a note for future reference - A potential hazard here is that SASS `@import` is dumb textual inclusion - so you could end up accidentally referencing a generically named variable like this in another file. A trick I have used in the client to avoid this is:

```scss
@at-root {
  $local-variable-name: value;

  // rest of the file here
}
```

Where `@at-root` starts a scope but does not alter the output CSS.